util/libssl.num

```diff
@@ -583,3 +583,26 @@ SSL_set_value_uint                      583	3_3_0	EXIST::FUNCTION:
 SSL_poll                                584	3_3_0	EXIST::FUNCTION:
 SSL_SESSION_get_time_ex                 585	3_3_0	EXIST::FUNCTION:
 SSL_SESSION_set_time_ex                 586	3_3_0	EXIST::FUNCTION:
+SSL_ech_set1_echconfig                  ?	3_4_0	EXIST::FUNCTION:ECH
+SSL_CTX_ech_set1_echconfig              ?	3_4_0	EXIST::FUNCTION:ECH
+SSL_ech_set_server_names                ?	3_4_0	EXIST::FUNCTION:ECH
+SSL_ech_set_outer_server_name           ?	3_4_0	EXIST::FUNCTION:ECH
+OSSL_ECH_INFO_free                      ?	3_4_0	EXIST::FUNCTION:ECH
+OSSL_ECH_INFO_print                     ?	3_4_0	EXIST::FUNCTION:ECH
+SSL_ech_get_info                        ?	3_4_0	EXIST::FUNCTION:ECH
+SSL_ech_reduce                          ?	3_4_0	EXIST::FUNCTION:ECH
+SSL_ech_get_status                      ?	3_4_0	EXIST::FUNCTION:ECH
+SSL_ech_set_grease_suite                ?	3_4_0	EXIST::FUNCTION:ECH
+SSL_ech_set_grease_type                 ?	3_4_0	EXIST::FUNCTION:ECH
+SSL_ech_set_callback                    ?	3_4_0	EXIST::FUNCTION:ECH
+SSL_ech_get_retry_config                ?	3_4_0	EXIST::FUNCTION:ECH
+SSL_CTX_ech_set_outer_alpn_protos       ?	3_4_0	EXIST::FUNCTION:ECH
+SSL_CTX_ech_server_enable_file          ?	3_4_0	EXIST::FUNCTION:ECH
+SSL_CTX_ech_server_enable_buffer        ?	3_4_0	EXIST::FUNCTION:ECH
+SSL_CTX_ech_server_enable_dir           ?	3_4_0	EXIST::FUNCTION:ECH
+SSL_CTX_ech_server_get_key_status       ?	3_4_0	EXIST::FUNCTION:ECH
+SSL_CTX_ech_server_flush_keys           ?	3_4_0	EXIST::FUNCTION:ECH
+SSL_CTX_ech_raw_decrypt                 ?	3_4_0	EXIST::FUNCTION:ECH
+SSL_CTX_ech_set_callback                ?	3_4_0	EXIST::FUNCTION:ECH
+ossl_ech_make_echconfig                 ?	3_4_0	EXIST::FUNCTION:ECH
```

Comment:
`ossl_...` is the prefix for internals methods, not for public ones.

Reply:
sure (though indentation in that file seems all over the place)